### PR TITLE
feat: Search Screens

### PR DESCRIPTION
**Pull request by girishsawant999**

### CODE REVIEW

File: www/src/components/common/Typography/typography.module.css [hunk 1]
```diff
@@ -32,3 +38,7 @@
   color: #9b939f;
   line-height: 1.83;
 }
+
+.bold {
```
Comment (Pranav2612000) [inline www/src/components/common/Typography/typography.module.css:42]: Use the heading type for extraboldManrope font

File: www/src/components/containers/search/components/SearchResultCountryList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import Img from 'components/common/img';
```
Comment (Pranav2612000) [inline www/src/components/containers/search/components/SearchResultCountryList/index.tsx:1]: Can a single component be used for this and CountryList?

Comment (girishsawant999) [reply www/src/components/containers/search/components/SearchResultCountryList/index.tsx:1]: I'll try for it

File: www/src/components/common/Typography/typography.module.css (hunk 1 above)
Comment (girishsawant999) [reply www/src/components/common/Typography/typography.module.css:42]: I'll change that

**Comment by girishsawant999** [comment]
Providing new PR

**Pull request closed without merging**